plugins/UM3NetworkPrinting/src/DiscoverUM3Action.py

```diff
@@ -105,63 +108,27 @@ def foundDevices(self):
         else:
             return []
 
-    @pyqtSlot(str)
-    def setGroupName(self, group_name: str) -> None:
-        Logger.log("d", "Attempting to set the group name of the active machine to %s", group_name)
-        global_container_stack = CuraApplication.getInstance().getGlobalContainerStack()
-        if global_container_stack:
-            # Update a GlobalStacks in the same group with the new group name.
-            group_id = global_container_stack.getMetaDataEntry("group_id")
-            machine_manager = CuraApplication.getInstance().getMachineManager()
-            for machine in machine_manager.getMachinesInGroup(group_id):
-                machine.setMetaDataEntry("group_name", group_name)
-
-            # Set the default value for "hidden", which is used when you have a group with multiple types of printers
-            global_container_stack.setMetaDataEntry("hidden", False)
-
+    @pyqtSlot()
+    def refreshConnections(self) -> None:
         if self._network_plugin:
-            # Ensure that the connection states are refreshed.
             self._network_plugin.refreshConnections()
 
-    # Associates the currently active machine with the given printer device. The network connection information will be
-    # stored into the metadata of the currently active machine.
-    @pyqtSlot(QObject)
-    def associateActiveMachineWithPrinterDevice(self, printer_device: Optional["PrinterOutputDevice"]) -> None:
-        if self._network_plugin:
-            self._network_plugin.associateActiveMachineWithPrinterDevice(printer_device)
-
-    @pyqtSlot(result = str)
-    def getStoredKey(self) -> str:
-        global_container_stack = CuraApplication.getInstance().getGlobalContainerStack()
-        if global_container_stack:
-            meta_data = global_container_stack.getMetaData()
-            if "um_network_key" in meta_data:
-                return global_container_stack.getMetaDataEntry("um_network_key")
-
-        return ""
-
+    # TODO: Improve naming
+    # TODO: CHANGE TO HOSTNAME
     @pyqtSlot(result = str)
     def getLastManualEntryKey(self) -> str:
         if self._network_plugin:
             return self._network_plugin.getLastManualDevice()
         return ""
 
+    # TODO: Better naming needed. Exists where? On the current machine? On all machines?
+    # TODO: CHANGE TO HOSTNAME
```

Comment:
Do these?

Reply:
No. That's the next step.

Reply:
It's best to not merge it until the feature is done, lest it be decided that the other work gets cancelled. Except if there is a good reason why it's not done yet (as in, it gets blocked by something). You're creating technical debt otherwise.